global.json

```
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "9.0.103",
+    "dotnet": "9.0.106",
```

Should this be updated to match the dogfood? I think so just so we're building on tip:
```suggestion
    "dotnet": "9.0.301",
```